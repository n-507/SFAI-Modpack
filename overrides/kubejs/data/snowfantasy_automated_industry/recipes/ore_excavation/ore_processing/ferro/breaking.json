{
  "type": "create:pressing",
  "ingredients": [
    {
      "item": "kubejs:rocky_ferro_batch"
    }
  ],
  "results": [
    {
      "item": "kubejs:dirty_ferro_batch"
    },
    {
      "chance": 0.5,
      "item": "minecraft:gravel" 
    }
  ]
}