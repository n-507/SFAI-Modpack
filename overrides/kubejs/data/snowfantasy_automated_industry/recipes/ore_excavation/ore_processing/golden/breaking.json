{
  "type": "create:pressing",
  "ingredients": [
    {
      "item": "kubejs:rocky_golden_batch"
    }
  ],
  "results": [
    {
      "item": "kubejs:dirty_golden_batch"
    },
    {
      "chance": 0.5,
      "item": "minecraft:gravel" 
    }
  ]
}